tests/integration/targets/aws_s3/tasks/main.yml

```diff
@@ -526,6 +534,49 @@
               - result is not changed
       when: ansible_system == 'Linux' or ansible_distribution == 'MacOSX'
 
+    - name: make a bucket with the bucket-owner-full-control ACL
+      s3_bucket:
+        name: "{{ bucket_name_acl }}"
+        state: present
+        policy: "{{ lookup('template', 'policy.json.j2') }}"
+        requester_pays: yes
+        versioning: yes
```

Comment:
Is versioning necessary to test this change?  We have a lambda function that runs in our CI (https://github.com/mattclay/aws-terminator) that cleans up any stray resources if they aren't properly removed in a test run and the versioned objects here aren't being removed, which is also preventing bucket removal.

Reply:
No it's not necessary. I removed the versioning and requester_pays parameter options